bindata/network/ovn-kubernetes/ovnkube-node.yaml

```diff
@@ -261,6 +286,11 @@ spec:
             gw_interface_flag="--exgw-interface=br-ex1"
           fi
 
+          node_mgmt_port_netdev_flags=
+          if [[ -n "${OVNKUBE_NODE_MGMT_PORT_NETDEV}" ]] ; then
```

Comment:
nothing seems to ever set this? Is this part not implemented yet?

Reply:
We set it via [configmap-override-cno-env.yaml](https://github.com/zshi-redhat/bluefield-demo/blob/a7cf4aff5c976fea8d9ce6ed1da3468805b9f3ea/manifests/configmap-override-cno-env.yaml), which is documented in the [Order of Installation Operation](https://docs.google.com/document/d/1m3xKFERxI_teUbln3dMVPY_xJ5hPLT3xM6JG_hFNZmI/edit#heading=h.7axwjok4nzxb) section, Step 8, of the "Pod Networking in a DPU Cluster" document. This is probably something that can automated later. In early development, the VF name changed depending on kernel version loaded.